mcts/config.json

```diff
@@ -9,9 +9,6 @@
                                       "%(temp_dir)s/omni_diff_report.html"]}],
             ["webapi", {"cmd": "webapirunner",
                         "run_args": ["--version=%(version)s"]}],
-            ["web-platform-tests", {"cmd": "wptrunner",
```

Comment:
Is it possible to still comment or log it elsewhere so I can remember this and modify accordingly later.